add guide to use Sentry with SolidJS (#8635)
getsentry/sentry-docs: src/platform-includes/getting-started-verify/javascript.solid.mdx

```javascript
<button
  type="button"
  onClick={() => {
    throw new Error("Sentry Frontend Error");
  }}
>
  Throw error
</button>
```

> This snippet adds a button that throws an error in any Solid component you choose. It is useful for testing. It is recommended to base wrap your app within an Error Boundary:

Reviewer: ```suggestion
This snippet adds a button that throws an error in any Solid component you choose. It's useful for testing. It's recommended to base wrap your app within an Error Boundary:
```
